pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -53,6 +54,7 @@ mkDerivation rec {
     qtgraphicaleffects
     qtwebsockets
     sqlite
+    inkscape
```

Comment:
If the icons are generated with inkscape this need to go to nativeBuildInputs.

Reply:
Why is that, may you elaborate please? i dont quite understand all the differences between the available build inputs.

Reply:
Good catch by @SuperSandro2000.

Inkscape should indeed only be required for building nextcloud-client, not for running the built nextcloud-client.

Reply:
See: https://discourse.nixos.org/t/use-buildinputs-or-nativebuildinputs-for-nix-shell/8464

Reply:
Oh i see, thanks a lot guys!